common/dynasties/2000_orc.txt

```diff
@@ -1,819 +1,3068 @@
-﻿# blackrock
+﻿### blackrock
 2000 = {
```

Comment:
It has no CoA.
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

Reply:
Done in 04d4183 by added a father for blackhand